src/main/java/org/gaul/s3proxy/S3ErrorCode.java

```diff
@@ -79,12 +79,17 @@
             "At least one of the preconditions you specified did not hold."),
     REQUEST_TIME_TOO_SKEWED(HttpServletResponse.SC_FORBIDDEN, "Forbidden"),
     REQUEST_TIMEOUT(HttpServletResponse.SC_BAD_REQUEST, "Bad Request"),
-    SIGNATURE_DOES_NOT_MATCH(HttpServletResponse.SC_FORBIDDEN, "Forbidden");
+    SIGNATURE_DOES_NOT_MATCH(HttpServletResponse.SC_FORBIDDEN, "Forbidden"),
+    XMAZCONTENTSHA256_MISMATCH(HttpServletResponse.SC_BAD_REQUEST,
+            "XAmzContentSHA256Mismatch");
 
     private final String errorCode;
     private final int httpStatusCode;
     private final String message;
 
+    // todo: XMAZCONTENTSHA256_MISMATCH cannot format as
+    // XAmzContentSHA256Mismatch
+    // use array like to record all this format, or add if condition!
```

Comment:
Not needed after addressing comment above.

Reply:
Done.